dev-python/gevent/metadata.xml

```diff
@@ -5,6 +5,9 @@
     <email>python@gentoo.org</email>
     <name>Python</name>
   </maintainer>
+  <use>
+    <flag name="events">Add dev-python/zope-event as a runtime dependency.</flag>
```

Comment:
The flag description should really say what it does, i.e. how would the user benefit from enabling it. Also `<pkg/>` around package name.

Reply:
Updated with a hopefully more informative description. I expect that this flag is likely to be used by other packages that depend on the gevent events system and want to set that as a required use flag (or at least, that is my use case).